data/nextstrain/yellow-fever/prM-E/pathogen.json

```
@@ -0,0 +1,52 @@
+{
+  "files": {
+    "reference": "reference.fasta",
+    "pathogenJson": "pathogen.json",
+    "genomeAnnotation": "genome_annotation.gff3",
+    "treeJson": "tree.json",
+    "examples": "sequences.fasta",
+    "readme": "README.md",
+    "changelog": "CHANGELOG.md"
+  },
+  "attributes": {
+    "name": "Yellow fever virus (YFV) prM-E region",
+    "reference name": "Asibi",
+    "reference accession": "AY640589.1"
+  },
+  "schemaVersion": "3.0.0",
+  "alignmentParams": {
+      "minSeedCover": 0.01,
+      "minLength": 500
```

```suggestion
      "minLength": 500
```

This seems unnecessarily restrictive - we should be able to use defaults here instead of setting an arbitrary high limit.